dotcom-rendering/src/lib/assets.ts

```diff
@@ -22,26 +22,37 @@ try {
  * @param {'PROD' | 'CODE' | undefined} stage the environment code is executing in
  * @returns {string}
  */
-const decideAssetOrigin = (stage: string | undefined): string => {
+const decideAssetOrigin = (
+	stage: string | undefined,
+	isDev: boolean,
+): string => {
 	switch (stage?.toUpperCase()) {
 		case 'PROD':
 			return 'https://assets.guim.co.uk/';
 		case 'CODE':
 			return 'https://assets-code.guim.co.uk/';
-		default:
-			return '/';
+		default: {
+			if (isDev) {
+				// Use absolute asset paths in development mode
+				// This is so paths are correct when treated as relative to Frontend
+				return 'http://localhost:3030/';
```

Comment:
I was unsure if there's a way to avoid hardcoding the dev server port here. Is there a way to obtain it from the webpack dev server config?

Reply:
you could require/import it like the mainifest.json above perhaps?

Reply:
This is a neat idea but I'm actually going to leave it, for now at least. I think it might be worth just keeping it as simple as possible.